pet.py

```diff
@@ -0,0 +1,63 @@
+import datetime
+
+# Define class
+class pet:
+    
+    def __init__(self, name, category, subtype, birthdate = None, colors = None, has_tail = False, has_wings = False, num_of_legs = 4, last_fed = datetime.date.today(), deathdate = None):
+        self.name = name
+        self.category = category
+        self.subtype = subtype
+        self.birthdate = birthdate
+        self.colors = colors
+        self.has_tail = has_tail
+        self.has_wings = has_wings
+        self.num_of_legs = num_of_legs
+        self.last_fed = last_fed
+        self.deathdate = deathdate
+
+    def getName(self):
+        self.name = input("What is your pet's name?:  ")
+
+    def getCategory(self):
+        self.category = input("What is the category (mammal, reptile, fish, etc...) that your pet falls in?:  ")
```

Comment:
Remove all the user input from the class.

This is more for the idea of separation of duties or isolating functionality.  I don't know how to explain why well.

The idea is the pet class should only define what a pet is, and information about this specific pet.  The methods allow you to send information into the pet class (set), or get information out of the pet class (get).  There are other methods that will perform an action, "died" and "feed" are examples where you are performing an action.